functions/package.json

```diff
@@ -4,6 +4,7 @@
     "lint": "eslint --ext .js,.ts .",
     "lint:fix": "eslint '*/**/*.{js,ts}' --quiet --fix",
     "build": "tsc",
+    "test": "tsc && mocha --reporter spec",
```

Comment:
considering CRA uses Jest I think we should keep it the same instead of bringing in Mocha? I'm not familiar with Mocha if it provides anything useful for testing cloud functions but.

Reply:
I would love to use jest to test the cloud functions, however the firebase cloud function tutorial only illustrates how to test with mocha and there are some specific things in the tests done with mocha which would not trivially translate to Jest. I tried to use Jest instead initially but gave up after a lot of troubles. 

I think this is more of a nicety than necessity and isn't a useful place to spend time